.travis.yml

```diff
@@ -46,6 +48,7 @@ notifications:
     skip_join: true
 
 before_deploy:
+ - apt-get download nsis nsis-common
```

Comment:
You can add the markdown package back again as well, and remove the download of the zip in `package-all.sh`